### PR TITLE
refactor: move to more supported react markdown package

### PR DESCRIPTION
**Pull request by crookse**
Introduced because `markdown-to-jsx` sucks when `deno fmt` is used. `react-markdown` works well.

Also introduces syntax highlighting for git diffs:

Before

Screen Shot 2021-12-04 at 8 24 36 PM

After

Screen Shot 2021-12-04 at 8 24 01 PM

### CODE REVIEW

**Comment by crookse** [comment]
converting this to draft... fkn styles are still messed up

File: src/components/Markdown.jsx [hunk 1]
```diff
@@ -1,14 +1,55 @@
 //
-// This file only contains exported styled components. These components are used
-// to replace the default components in the "markdown-to-jsx" package. See
-// the `[...path_params].js` file to see how they are used.
+// This file only contains exported styled components. These components are
+// used to replace the default components in our markdown package that we use
+// in `[...path_params].jsx`.
 //
+import React from "react";
+
```
Comment (ebebbington) [inline src/components/Markdown.jsx:7]: Noticed this is declared in 2 places (same as the function below), any way we can declare them one and re use?

**Review by ebebbington — APPROVED** [review]
Just that one comment but fine with me :)

File: src/components/Markdown.jsx (hunk 1 above)
Comment (crookse) [reply src/components/Markdown.jsx:7]: woops! i originally had it in the path_params component just to test it out and forgot to take it out after i moved it to the Markdown component. done! 83ccdd7

**Pull request merged**